tests/unit/src/kakarot/precompiles/test_ripemd160.py

```diff
@@ -19,16 +20,25 @@ async def ripemd160(
 @pytest.mark.asyncio
 class TestRIPEMD160:
     @pytest.mark.parametrize(
-        "msg, expected_result", [
-            ("a", "0000000000000000000000000bdc9d2d256b3ee9daae347be6f4dc835a467ffe"),
-            (
-                "abcdbcdecdefdefgefghfghighijhijkijkljklmklmnlmnomnopnopqabcdbcdecdefdefgefghfghighijhijkijkljklmklmnlmnomnopnopq",
-                "00000000000000000000000069a155bddf855b0973a0791d5b7a3326fb83e163"
-            ),
-        ]
+        "msg_len",
+            [random.randint(1, 300) for _ in range(3)]
     )
-    async def test_ripemd160_should_return_correct_hash(self, ripemd160, msg, expected_result):
-        ascii_msg = [ord(char) for char in msg]
-        expected_result_byte_array = list(bytes.fromhex(expected_result))
-        hash =  (await ripemd160.test__ripemd160(ascii_msg).call()).result[0]
-        assert hash == expected_result_byte_array
+    async def test_ripemd160_should_return_correct_hash(self, ripemd160, msg_len):
+        ripemd160_crypto = RIPEMD160.new()
+
+        # Build message to be hashed
+        message = "".join((chr(random.randint(0, 255))) for _ in range(msg_len))
+        message_bytes = bytes(message, encoding='utf8')
```

Comment:
what about a `message_bytes = bytearray([random.randint(0,  255) for _ in range(len))`